Dependencies/UrlLib/Source/Unix/UrlRequest.cpp

```diff
@@ -174,7 +174,61 @@ namespace UrlLib
             if (curl)
             {
                 data.clear();
-                curl_easy_setopt(curl, CURLOPT_URL, m_url.data());
+
+                // libCurl doesn't escape url strings automatically.
```

Comment:
nit: it might be good to put this in a private helper function to make the code easier to read.

Reply:
Will update in my next PR.